storage/operation/cluster.go

```diff
@@ -29,12 +55,45 @@ func LookupClusterBlockHeight(r storage.Reader, clusterID flow.ChainID, height u
 	return RetrieveByKey(r, MakePrefix(codeFinalizedCluster, clusterID, height), blockID)
 }
 
-// UpsertClusterFinalizedHeight updates (overwrites!) the latest finalized cluster block height for the given cluster.
-func UpsertClusterFinalizedHeight(lctx lockctx.Proof, w storage.Writer, clusterID flow.ChainID, number uint64) error {
+func BootstrapClusterFinalizedHeight(lctx lockctx.Proof, rw storage.ReaderBatchWriter, clusterID flow.ChainID, number uint64) error {
```

Comment:
Based on our [coding conventions](https://github.com/onflow/flow-go/blob/master/CodingConventions.md), exported function should have at least some basic documentation including error returns!  Furthermore, documentation of the storage lock is missing.

```suggestion
// BootstrapClusterFinalizedHeight initializes the latest finalized cluster block height for the given cluster.
//
// CAUTION:
//   - This function is intended to be called during bootstrapping only. It expects that the height of the latest
//     known finalized cluster block has not yet been persisted.
//   - Confirming that no value is already stored and the subsequent write must be atomic to prevent data corruption.
//     Therefore, the caller must acquire the [storage.LockInsertOrFinalizeClusterBlock] and hold it until the database
//     write has been committed.
//
// No error returns expected during normal operations.
func BootstrapClusterFinalizedHeight(lctx lockctx.Proof, rw storage.ReaderBatchWriter, clusterID flow.ChainID, number uint64) error {
```